src/drivers/intel/ssp/mn.c

```diff
@@ -326,8 +326,14 @@ static bool find_mn(uint32_t freq, uint32_t bclk,
 static int find_bclk_source(uint32_t bclk,
 			    uint32_t *scr_div, uint32_t *m, uint32_t *n)
 {
+	struct mn *mn = mn_get();
 	int i;
 
+	/* check if we can use MCLK source clock */
+	if (is_mclk_source_in_use() &&
+	    find_mn(ssp_freq[mn->mclk_source_clock].freq, bclk, scr_div, m, n))
```

Comment:
Can we have a trace_error if we cant find the same clock source. That way it should be obvious from the logs.

Reply:
Ok, added warning if find_mn() failed